src/main/scala/ae/network/migration/spark/DataHandling/DataReader.scala

```diff
@@ -0,0 +1,21 @@
+package ae.network.migration.spark.DataHandling
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+
+object DataReader {
+
+
+  def read(spark: SparkSession, path: String): DataFrame = {
```

Comment:
Could you rename it to what type of reading you are doing, for example `readCsv`